x-pack/plugins/lens/public/xy_visualization/visualization.tsx

```diff
@@ -305,6 +435,30 @@ export const getXyVisualization = ({
       newLayer.splitAccessor = columnId;
     }
 
+    if (newLayer.layerType === layerTypes.THRESHOLD) {
+      newLayer.accessors = [...newLayer.accessors.filter((a) => a !== columnId), columnId];
+      const hasYConfig = newLayer.yConfig?.some(({ forAccessor }) => forAccessor === columnId);
+      if (!hasYConfig) {
+        newLayer.yConfig = [
+          ...(newLayer.yConfig || []),
+          // TODO: move this
```

Comment:
what';s this comment about? 🤔

Reply:
The styling configuration should be probably moved elsewhere. This method should be only about set accessors